config/routes.rb

```diff
@@ -64,6 +64,7 @@
     resources :courses do
       resources :announcements
       resources :achievements
+      get 'settings' => 'course_settings#index', as: :settings
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.